ui/theme/theme.go

```diff
@@ -205,11 +205,24 @@ func (m *MyTheme) Icon(name fyne.ThemeIconName) fyne.Resource {
 
 // Returns a map [themeFileName] -> displayName
 func (m *MyTheme) ListThemeFiles() map[string]string {
-	files, _ := filepath.Glob(m.themeFileDir + "/*.toml")
+	// Use filepath.Join to create a cross-platform file path
+	pattern := filepath.Join(m.themeFileDir, "/*.toml")
+	files, err := filepath.Glob(pattern)
+	if err != nil {
+		log.Fatalf("Failed to glob files: %v", err)
```

Comment:
log.Fatalf will crash the app, which we don't want even if it fails here. It's better to just log.Printf and return nil